### ISSUE TITLE
Dropout not working in fa-train

### ISSUE THREAD

**Issue by jcreinhold** [jcreinhold/synthnn#1]
At least when 1-cycle policy is enabled, dropout > 0 causes backprop to fail. It states that an inplace operation has changed a value needed for the gradient.

**Comment by jcreinhold** [comment]
Set inplace=False in dropout blocks solved the issue
